Proto/cg.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
@dsyme, @KevinRansom some OCamlers are being creative nowadays:

https://github.com/BYVoid/Batsh

Somehow, I think we should stick to our guns for the future (that is doing FAKE/.fsx scripts) but maybe shell script lovers want to chime in :smile: